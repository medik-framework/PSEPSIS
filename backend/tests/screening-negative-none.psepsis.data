{
 "hr"                 : 150,
 "bpSys"              : null,
 "temp"               : 37,
 "pulseQuality"       : null,
 "mentalStatus"       : "normal",
 "capillaryRefill"    : 2,
 "skinColor"          : "normal",
 "highRiskConditions" : 0
}

