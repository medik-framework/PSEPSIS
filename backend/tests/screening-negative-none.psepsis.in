[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartScreening",
    "eventArgs": []
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [500],
    "waitFor":
     {
       "name": "Instruct",
       "args": ["get age"]
     }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "WeightEntered",
    "eventArgs": [10],
    "waitFor":
      {
        "name": "Instruct",
        "args": ["get weight"]
      }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "HighRiskConditionsEntered",
    "eventArgs": [0],
    "waitFor":
      {
        "name": "Instruct",
        "args": ["get high risk conditions"]
      }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmMeasurementsObtained",
    "eventArgs": [],
    "waitFor":
      {
        "name": "Instruct",
        "args": ["get 3 bucket measurements"]
      }
  },
  {
    "action": "exit",
    "waitFor":
     {
       "name": "SepsisDiagnosis"
     }
  }
]
