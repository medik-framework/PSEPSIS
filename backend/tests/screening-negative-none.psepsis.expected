[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get age"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get weight"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmWeightEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get high risk conditions"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmHighRiskConditionsEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get 3 buckets measurements"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "hr"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "bpSys"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "temp"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "pulseQuality"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "mentalStatus"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "capillaryRefill"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "skinColor"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "highRiskConditions"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "DisplayDiagnosis",
    "args": [
      false
    ]
  }
]
